### ISSUE TITLE
UnicodeEncodeError in django_manage module when attempting to run custom django task

### ISSUE THREAD

**Issue by vinigfer** [ansible/ansible#22834]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
Module: django_manage

##### ANSIBLE VERSION
```
ansible 2.2.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

##### CONFIGURATION
Not sure if it helps, but this is the content from /etc/ansible/hosts:
```
# Application servers
[AnsibleServer]
127.0.0.1

# Variables that will be applied to all servers
[AnsibleServer:vars]
ansible_ssh_user=telessaude
ansible_port=3022
ansible_ssh_private_key_file=~/.ssh/id_rsa
```

##### OS / ENVIRONMENT
Running ansible from (physical machine, 8GB RAM, i5-4570 CPU, 64-bit):
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.2 LTS
Release:	16.04
Codename:	xenial

Managing a VM with the Ubuntu 16.04.02, but 1 vCPU and 2GB RAM

##### SUMMARY
I'm having an UnicodeEncodeError in django_manage module when attempting to run a custom django task. Almost sure this happens due to an accented letter.

##### STEPS TO REPRODUCE
Run ansible task with the command
`ansible-playbook playbook.yml --tags init_permissions`

If I ssh into the VM, enter the django project folder, and run this command

`python manage.py init_permissions`

I get the following output, with no errors:

```
...
Solicitante: servicos.avaliacaosolicitacao.ver_avaliacao_solicitacao
Solicitante: servicos.sugestaohorario.pode_ver_sugestao_agendamento
Solicitante: servicos.sugestaohorario.pode_sugerir_horario_agendamento
Solicitante: perfis.perfil.acessar_agenda
Técnico: servicos.solicitacao.acessar_caixa_entrada
Técnico: servicos.solicitacao.acessar_caixa_saida
Técnico: servicos.solicitacao.acessar_caixa_final
...
```

but if I try to run the same command using Ansible using a task, like the one below, I get the errors.

```yaml
    - name: Initiating permissions
      django_manage:
        command: "init_permissions"
        app_path: "{{ destination_folder }}"
      tags:
        - init_permissions
```

##### EXPECTED RESULTS
No errors (something such as ok: "[127.0.0.1]")

##### ACTUAL RESULTS
...
Solicitante: servicos.sugestaohorario.pode_ver_sugestao_agendamento\n
Solicitante: servicos.sugestaohorario.pode_sugerir_horario_agendamento\n
Solicitante: perfis.perfil.acessar_agenda\n\n:
stderr: /home/telessaude/telessaude_branch_master/core/management/commands/init_permissions.py:144: RemovedInDjango19Warning: django.db.models.get_model is deprecated.\n
  model = get_model(app_name, model_name)\n\n
/usr/local/lib/python2.7/dist-packages/django/db/models/__init__.py:55: RemovedInDjango19Warning: The utilities in django.db.models.loading are deprecated in favor of the new application loading system.\n
  from . import loading\n\nTraceback (most recent call last):\n
  File \"./manage.py\", line 25, in <module>\n
    execute_from_command_line(sys.argv)\n
  File \"/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py\", line 354, in execute_from_command_line\n
    utility.execute()\n
  File \"/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py\", line 346, in execute\n
    self.fetch_command(subcommand).run_from_argv(self.argv)\n
  File \"/usr/local/lib/python2.7/dist-packages/django/core/management/base.py\", line 394, in run_from_argv\n
    self.execute(*args, **cmd_options)\n
  File \"/usr/local/lib/python2.7/dist-packages/django/core/management/base.py\", line 445, in execute\n
    output = self.handle(*args, **options)\n
  File \"/home/telessaude/telessaude_branch_master/core/management/commands/init_permissions.py\", line 165, in handle\n
    self.parse_permissions(group, permissions)\n
  File \"/home/telessaude/telessaude_branch_master/core/management/commands/init_permissions.py\", line 185, in parse_permissions\n
    self.add_permission(group, permission_name, app_name, model_name)\n
  File \"/home/telessaude/telessaude_branch_master/core/management/commands/init_permissions.py\", line 196, in add_permission\n
    print(u'{0}: {1}.{2}.{3}'.format(group, app_name, model_name, permission_name))\n
UnicodeEncodeError: 'ascii' codec can't encode character u'\\xe9' in position 1: ordinal not in range(128)\n
", "path": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games", "state": "absent", "syspath": ["/tmp/ansible_UEYQXN", "/tmp/ansible_UEYQXN/ansible_modlib.zip", "/tmp/ansible_UEYQXN/ansible_modlib.zip", "/usr/lib/python2.7", "/usr/lib/python2.7/plat-x86_64-linux-gnu", "/usr/lib/python2.7/lib-tk", "/usr/lib/python2.7/lib-old", "/usr/lib/python2.7/lib-dynload", "/usr/local/lib/python2.7/dist-packages", "/usr/lib/python2.7/dist-packages"]}

**Comment by ansibot** [comment]
cc @tastychutney @scottanderson42
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by mmclark** [comment]
@vinigfer it looks like this may be an issue in your custom command.  I spun up a 64-bit  Ubuntu 16.04 VM with a simple custom command using a similar print statement with a unicode character.  It's failing both locally (ssh running python manage.py directly) and using the ansible django_manage module.  Can you provide some more info on how the command is succeeding when you've ssh'd into the VM?

This is the error I receive in both cases:

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/vagrant/venv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/vagrant/venv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/vagrant/venv/local/lib/python2.7/site-packages/django/core/management/base.py", line 390, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/vagrant/venv/local/lib/python2.7/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(*args, **options)
  File "/vagrant/testsite/hello/management/commands/sayhello.py", line 10, in handle
    print(u'Hello there, {0}!'.format('Th\xe9o'))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 2: ordinal not in range(128)
```

**Comment by abadger** [comment]
needs_info

**Comment by ansibot** [comment]
@vinigfer This issue is waiting for your response. Please respond or the issue will be closed.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@vinigfer This issue is waiting for your response. Please respond or the issue will be closed.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@vinigfer You have not responded to information requests in this issue so we will assume it no longer affects you. If you are still interested in this, please create a new issue with the requested information.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ralffunk** [comment]
I don't know if it is right for me to comment on closed issues, so feel free to point me in the right direction; but I want to share my solution, but don't know if it warrants for me to oppen a new issue.

I had the same symptoms described in this issue (small difference: I was using ubuntu server 18.04) and in my case I only needed to add an environment variable to the task that throwed the error. By adding the following I got it working:

```
  environment:
    LC_ALL: 'en_US.UTF-8'

```
I did not find the answer here, but a kind user on StackOverflow suggested this when I [asked about it](https://stackoverflow.com/q/53557236/9225671).
